lib/org/openpsa/invoices/handler/invoice/view.php

```diff
@@ -96,6 +96,11 @@ private function populate_toolbar()
                  && intval($billing_data->sendingoption) == 2) {
                 $buttons[] = $this->build_button('send_by_mail', 'paper-plane');
             }
+
+            $button = $this->build_button('create_reminder', 'file-pdf-o');
+            $pdf_helper = new org_openpsa_invoices_invoice_pdf($this->invoice);
+            $button[MIDCOM_TOOLBAR_OPTIONS] = $pdf_helper->get_button_options();
+            $buttons[] = $button;
```

Comment:
this button should only be available if the invoice is overdue

Reply:
i'va made a mistake at this point. i misinterpreted a int as a datetime value